dodiscover/replearning/gin.py

```diff
@@ -0,0 +1,115 @@
+"""
+Wrapper for the GIN latent variable causal discovery algorithm in causal-learn
+"""
+
+from pandas import DataFrame
+
+from causallearn.search.HiddenCausal.GIN.GIN import GIN as GIN_
```

Comment:
In order to make causal-learn an optional dependency, this import statement should maybe go inside `fit()`.

Reply:
@adam2392 [6380da3](https://github.com/py-why/dodiscover/pull/72/commits/6380da386142732c6c3300dbec655fbd731c37e8)

Reply:
[fed4d36](https://github.com/py-why/dodiscover/pull/72/commits/fed4d36e23b8c18892bf7a6f5d6ec4a6d82ee7ec)